packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  getFocusStyle,
+  HighContrastSelector
+} from '../../Styling';
+import {
+  ILinkStyleProps,
+  ILinkStyles
+} from './Link.types';
+
+export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
+  const { disabled, theme } = props;
```

Comment:
We should also className to the props so people can add there own classNames to the root element.

Reply:
This one I intentionally left out, because the existing component never had support for `className`. I think we should maintain existing functionality but not add support for a method a styling that we're likely to remove in the future.